docs/spec/spec.md

```diff
@@ -896,6 +896,8 @@ enum Direction {
 }
 ```
 
+>**Note:** While it is possible to assign a `string` value to an `enum` member in the Ballerina language, the GraphQL specification does not support this behaviour. Therefore, it's not recommended to use `string` values for `enum` members when defining an `enum` type for use in a `graphql:Service`. Using `string` values for `enum` members may result in an invalid generated schema.
```

Comment:
Shall we also describe the default value behavior under section `4.5.3`? Basically how we generate the schema and best practices of defining default values.

Reply:
Updated with: [5ee0f31](https://github.com/ballerina-platform/module-ballerina-graphql/pull/1586/commits/5ee0f31d254490a35c7a58916997ea38db10e86e)